### PR TITLE
added official C++ logo from  (resized…

### PR DESCRIPTION
**Pull request by jsamuel1**
Sourced from https://github.com/isocpp/logos
Resized with Imagemagick:
`mogrify -resize 64x64 -background transparent -gravity Center -extent 64x64 -format png -strip -path . ~/Downloads/cpp_logo.png`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Buildkite emoji will be shown on both light or dark backgrounds, and at a small
 
 Emoji | Aliases
 ----- | -------
+<img src="img-buildkite-64/cpp.png" width="20" height="20" alt="cpp"/> | `:cpp: :cplusplus: :c++:`
```
Comment (ticky) [inline README.md]:
```suggestion
<img src="img-buildkite-64/cpp.png" width="20" height="20" alt="cpp"/> | `:cpp:`, `:cplusplus:`, `:c++:`
```

File: img-buildkite-64.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "cpp",
+    "image": "img-buildkite-64/cpp.png",
+    "category": "Buildkite",
+    "aliases": [
+    "cplusplus",
```
Comment (ticky) [inline img-buildkite-64.json]:
```suggestion
      "cplusplus",
```

File: img-buildkite-64.json [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+    "c++"
```
Comment (ticky) [inline img-buildkite-64.json]:
```suggestion
      "c++"
```

**Comment by jsamuel1** [comment]
> Like in #204, there's just some minor formatting stuff before this is ready to go! Thanks again for your contributions! :D

Thanks @ticky.   Committed your suggestions and resolved the merge conflict with master from the Jenkins addition.

**Pull request merged**